store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9262,14 +9262,20 @@ TODO: delete them permanently from here
 <attr id="2096" name="zimbraSieveFeatureVariablesEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.7.6" deprecatedSince="8.7.8">
   <globalConfigValue>FALSE</globalConfigValue>
   <desc>Whether to enable the Sieve "Variables" extension defined in RFC 5229 in the user-defined sieve rule.</desc>
-  <deprecateDesc>deprecated in favor of zimbraSieveRejectMailEnabled, which can be used at account level</deprecateDesc>
+  <deprecateDesc>deprecated as this attribute is not used anymore and variable feature is availabe to use for all.</deprecateDesc>
```

Comment:
Will update PR with "Variable feature is always enabled, hence this attribute has been deprecated"